ui/de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -353,8 +351,13 @@ public void onShareSelected(List<Analysis> selected) {
 
     @Override
     public void setCurrentFilter(AnalysisFilter filter) {
-       this.currentFilter = filter;
-       loadAnalyses(currentFilter);
+        if(filter == null) {
+            currentFilter =filter;
+            return;
```

Comment:
Does this mean that the analysis listing won't be refreshed if the user removes the current filter?